modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -90,6 +90,31 @@ message Product {
 
 You would want the compatibility setting to accommodate adding new fields without breakage. Adding an optional new field to a schema is inherently backward-compatible. New consumers can process events written with the new schema, and older consumers can ignore it. 
 
+== JSON schema 
```

Comment:
```suggestion
== JSON Schema 
```

Reply:
Looks like this was already updated @Deflaimun